client/components/footer/footer.test.js

```diff
@@ -7,6 +7,29 @@ import getConfig from "../../utils/get-config";
 import Footer from "./footer";
 
 const defaultConfig = getConfig("default");
+const footerLinks = [
+  {
+    text: {en: "link-1"},
+    url: "https://link-1.com",
+    authenticated: true,
```

Comment:
would be good to use real world examples like the one I suggested for the docs also in the tests, it makes the tests more readable and the code is less costly to maintain as a result